lib/rspec/support/ruby_features.rb

```diff
@@ -65,6 +65,25 @@ def supports_exception_cause?
         end
       end
 
+      if Ruby.mri? && RUBY_VERSION >= '1.9.0'
+        def ripper_supported?
+          true
+        end
+      elsif Ruby.jruby? &&
+            defined?(Gem::Version) &&
+            Gem::Version.new(JRUBY_VERSION) >= Gem::Version.new('1.7.14') &&
+            !JRUBY_VERSION.start_with?('9.')
```

Comment:
This conditional will return `false` if rubygems is not loaded.  It should not be a requirement that rubygems is loaded to use this feature.  Users may want to run `rspec` w/ the `--disable=gems` flag and that should work, IMO.  I think you can use our version checker instead:

https://github.com/rspec/rspec-support/blob/master/lib/rspec/support/version_checker.rb